pkg/arvo/sys/zuse.hoon

```diff
@@ -6367,7 +6375,7 @@
       (ifix [gal ban] ;~(plug name attr))
     ::                                                  ::  ++many:de-xml:html
     ++  many                                            ::  contents
-      (more (star comt) ;~(pose apex chrd))
+      (star ;~(sfix ;~(pose apex chrd cdat) (star comt)))
```

Comment:
The issue is how to allow comments between any of the items, right?  Could this be `;~(pfix (star comt) (star ;~(sfix ;~(pose apex chrd cdat) (star comt)))`?

Reply:
What was wrong with the original ++more formulation in that case?

Reply:
Thanks @philipcmonk that works. @ohAitch the original formulation failed to parse on `<elem><!-- comment --></elem>` and `<elem>text<!-- comment --></elem>`

Reply:
If you make this change, I'll approve this PR.

Reply:
Ah, so really it should be `(star ;~(pose apex chrd cdat comt))`, with `comt` parsing to `~`?

Reply:
That would leave around `~`s in the parsed result, which doesn't sound right.

Reply:
That makes sense, though if it's objectionable we should probably also be `zing`ing together adjacent chrd and cdat(or moving cdata handling inside chrd instead of being a toplevel node, rather)